### ISSUE TITLE
stopShimmer leaves ShimmerDrawable visible 

### ISSUE THREAD

**Issue by Glurt** [facebook/shimmer-android#85]
I might be using this library differently from how it was intended but I think this issue could be any easy fix.

I have my RecyclerView inside the ShimmerFrameLayout, when the view is first created RecyclerViewAdapter is given a list of `null` and the adapter represents each of the `null`s by creating placeholder ViewHolders, I then tell the ShimmerFrameLayout to start shimmering. 

Once content has arrived the nulls are replaced with actual data and the ViewHolders are populated, the shimmering is then stopped using `stopShimmer()`. The issue is that the `mShimmerDrawable` in ShimmerFrameLayout is still displayed meaning the content is masked. 

I can work around this by setting the shimmer to null when stopping it and then adding it again when starting like so:

```
shimmerLayout.stopShimmer()
shimmerLayout.setShimmer(null)
```

```
shimmerLayout.setShimmer(Shimmer.AlphaHighlightBuilder().build())
shimmerLayout.startShimmer()
```

Can this be handled internally by either hiding the drawable when stopping or through some other means?

**Comment by subrud** [comment]
I think the separate layout has to be provided for shimmer layout. Check this article https://medium.com/mindorks/android-design-shimmer-effect-fa7f74c68a93

**Comment by xiphirx** [comment]
`stopShimmer` stops the shimmering animation, if you want it completely gone, then you do need to `setShimmer(null)`. Adding `hideShimmer`, `showShimmer` methods seems fine, though.

**Comment by xiphirx** [comment]
PR for that is certainly welcome if you're up for it :)

**Comment by Glurt** [comment]
Closed as part of #86
